### PR TITLE
`AreaContainmentObject` and `BiomeTag` examples

### PR DESCRIPTION
**Pull request by BreadcrumbIsTaken**
Added examples for `AreaContainmentObject` tags and `BiomeTag` tags.

I didn't add mechs for `BiomeTags` yet as I am not currently able to test them. 

I also fixed up some new line things and de-dented `<AreaContainmentObject.spawnable_blocks>`.
:)
###### 🐊🐊🐊

### CODE REVIEW

File: plugin/src/main/java/com/denizenscript/denizen/objects/BiomeTag.java [hunk 1]
```diff
@@ -201,28 +209,37 @@ public static void registerTags() {
         // @mechanism BiomeTag.humidity
         // @description
         // Returns the humidity of this biome.
+        // @Example
+        // - narrate "Humidity is currently <player.location.biome.humidity>! So humid!"
```
Comment (mcmonkey4eva) [inline plugin/src/main/java/com/denizenscript/denizen/objects/BiomeTag.java]: Should specify an example of the expected result, eg `# If you're standing in a Plains biome this fills with '0.5'` or something

Comment (BreadcrumbIsTaken) [reply plugin/src/main/java/com/denizenscript/denizen/objects/BiomeTag.java]: I added this to a few other related tags. Like `<BiomeTag.downfall_type>`, `<BiomeTag.humidity>`, `<BiomeTag.temperature>`

File: plugin/src/main/java/com/denizenscript/denizen/objects/AreaContainmentObject.java [hunk 2]
```diff
@@ -96,6 +96,10 @@ static <T extends AreaContainmentObject> void registerTags(Class<T> type,  Objec
         // @description
         // Returns a cuboid approximately representing the maximal bounding box of the area (anything this cuboid does not contain, is also not contained by the area, but not vice versa).
         // For single-member CuboidTags, this tag returns a copy of the cuboid.
+        // @Example
+        // # Gets the cuboid's bounding box and plays a totem effect at the shell of the cuboid.
```
Comment (mcmonkey4eva) [inline plugin/src/main/java/com/denizenscript/denizen/objects/AreaContainmentObject.java]: This example makes no sense. A cuboid's bounding box is... just the cuboid.

File: plugin/src/main/java/com/denizenscript/denizen/objects/AreaContainmentObject.java [hunk 3]
```diff
@@ -107,6 +111,8 @@ static <T extends AreaContainmentObject> void registerTags(Class<T> type,  Objec
         // @returns WorldTag
         // @description
         // Returns the area's world.
+        // @Example
```
Comment (mcmonkey4eva) [inline plugin/src/main/java/com/denizenscript/denizen/objects/AreaContainmentObject.java]: For some reason meta norm is lowercase key names in tags (and capital for most other things, idk why, but, maintain the existing format)

File: plugin/src/main/java/com/denizenscript/denizen/objects/AreaContainmentObject.java [hunk 4]
```diff
@@ -117,6 +123,9 @@ static <T extends AreaContainmentObject> void registerTags(Class<T> type,  Objec
         // @returns ListTag(PlayerTag)
         // @description
         // Gets a list of all players currently within the area.
+        // @Example
+        // # Narrates a list of players' names that are within the area separated by a comma and a space.
```
Comment (mcmonkey4eva) [inline plugin/src/main/java/com/denizenscript/denizen/objects/AreaContainmentObject.java:126]: Give an example of what you'd expect this to actually output as. Like `bob, joe, steve`

File: plugin/src/main/java/com/denizenscript/denizen/objects/AreaContainmentObject.java [hunk 5]
```diff
@@ -133,6 +142,9 @@ static <T extends AreaContainmentObject> void registerTags(Class<T> type,  Objec
         // @returns ListTag(NPCTag)
         // @description
         // Gets a list of all NPCs currently within the area.
+        // @Example
+        // # Narrates a list of NPCs' names that are within the area separated by a comma and a space.
+        // - narrate "List of NPCs in 'my_cuboid': <cuboid[my_cuboid].npcs.parse[name].separated_by[, ]"
```
Comment (mcmonkey4eva) [inline plugin/src/main/java/com/denizenscript/denizen/objects/AreaContainmentObject.java]: This is missing the `>` at the end.

Make sure you actually test your examples.

File: plugin/src/main/java/com/denizenscript/denizen/objects/AreaContainmentObject.java [hunk 6]
```diff
@@ -152,6 +164,10 @@ static <T extends AreaContainmentObject> void registerTags(Class<T> type,  Objec
         // @returns ListTag(EntityTag)
         // @description
         // Gets a list of all entities currently within the area, with an optional search parameter for the entity.
+        // @Example
+        // # Narrates all the entities that match 'axolotl' within the area.
```
Comment (mcmonkey4eva) [inline plugin/src/main/java/com/denizenscript/denizen/objects/AreaContainmentObject.java]: Narrates an entity? wat?

File: plugin/src/main/java/com/denizenscript/denizen/objects/AreaContainmentObject.java [hunk 7]
```diff
@@ -189,6 +209,12 @@ static <T extends AreaContainmentObject> void registerTags(Class<T> type,  Objec
         // @returns ElementTag(Boolean)
         // @description
         // Returns a boolean indicating whether the specified location is inside this area.
+        // @Example
+        // # Checks to see if "my_cuboid" contains the player's location.
+        // - if <cuboid[my_cuboid].contains[<player.location>]:
```
Comment (mcmonkey4eva) [inline plugin/src/main/java/com/denizenscript/denizen/objects/AreaContainmentObject.java]: again, test your examples

File: plugin/src/main/java/com/denizenscript/denizen/objects/AreaContainmentObject.java [hunk 8]
```diff
@@ -245,6 +279,10 @@ static <T extends AreaContainmentObject> void registerTags(Class<T> type,  Objec
         // @description
         // Gets a list of all block locations with a specified flag within the area.
         // Searches the internal flag lists, rather than through all possible blocks.
+        // @Example
+        // # Narrates the locations of blocks that are flagged "my_flag"
+        // - foreach <cuboid[my_cuboid].blocks_flagged[my_flag]> as:block:
+        //      - narrate <[block]>
```
Comment (mcmonkey4eva) [inline plugin/src/main/java/com/denizenscript/denizen/objects/AreaContainmentObject.java]: This is narrating a raw object

File: plugin/src/main/java/com/denizenscript/denizen/objects/AreaContainmentObject.java [hunk 9]
```diff
@@ -256,6 +294,9 @@ static <T extends AreaContainmentObject> void registerTags(Class<T> type,  Objec
         // @description
         // Returns each block location on the 3D outer shell of the area.
         // This tag is useful for displaying particles or blocks to mark the boundary of the area.
+        // @Example
+        // # Plays the "TOTEM" effect in the shell of the area.
+        // - playeffect effect:TOTEM at:<cuboid[my_cuboid].shell>
```
Comment (mcmonkey4eva) [inline plugin/src/main/java/com/denizenscript/denizen/objects/AreaContainmentObject.java]: Again, testing, this time for the reason that the `playeffect` command has a bunch of weirdities and you've hit 2 or 3 of them in this example that you'd see if you tried it.

File: plugin/src/main/java/com/denizenscript/denizen/objects/BiomeTag.java [hunk 10]
```diff
@@ -180,6 +180,9 @@ public static void registerTags() {
         // @description
         // Returns this biome's downfall type for when a world has weather.
         // This can be RAIN, SNOW, or NONE.
+        // @Example
+        // # When in a Plains biome, this fills with 'RAIN'.
```
Comment (mcmonkey4eva) [inline plugin/src/main/java/com/denizenscript/denizen/objects/BiomeTag.java]: ` When in ` doesn't make sense as the example is explicitly only `plains`

File: plugin/src/main/java/com/denizenscript/denizen/objects/AreaContainmentObject.java [hunk 11]
```diff
@@ -96,6 +96,9 @@ static <T extends AreaContainmentObject> void registerTags(Class<T> type,  Objec
         // @description
         // Returns a cuboid approximately representing the maximal bounding box of the area (anything this cuboid does not contain, is also not contained by the area, but not vice versa).
         // For single-member CuboidTags, this tag returns a copy of the cuboid.
+        // @example
+        // # Narrates the cuboids bounding box, which is just the cuboid.
+        // - narrate <cuboid[my_cuboid].bounding_box>
```
Comment (mcmonkey4eva) [inline plugin/src/main/java/com/denizenscript/denizen/objects/AreaContainmentObject.java:101]: This is a bad example.

File: plugin/src/main/java/com/denizenscript/denizen/objects/AreaContainmentObject.java [hunk 12]
```diff
@@ -200,6 +227,10 @@ static <T extends AreaContainmentObject> void registerTags(Class<T> type,  Objec
         // @description
         // Returns each block location within the area.
         // Optionally, specify a material matcher to only return locations with that block type.
+        // @example
+        // # Narrates the locations of blocks that match "*planks" within the area.
+        // - foreach <cuboid[my_cuboid].blocks[*planks]> as:plank:
+        //      - narrate <[plank]>
```
Comment (mcmonkey4eva) [inline plugin/src/main/java/com/denizenscript/denizen/objects/AreaContainmentObject.java:233]: This is narrating a raw location. Should use a more appropriate example behavior, perhaps a `debugblock`

**Pull request merged**